data/json/mapgen_palettes/house_general_palette.json

```diff
@@ -0,0 +1,185 @@
+[
+  {
+    "type": "palette",
+    "id": "house_palette",
```

Comment:
Is this intended to be a standard domestic palette?

Reply:
That's my hope.  I will definitely use this in every house I rehab, and I'm considering recommending it in some beginner mapgen documentation I plan on writing.  

Any nested house I make, will use my other palette set, but this is a good setup for all houses that have static interiors.  Hopefully, it'll gain wide use and we can manage future house loot changes by updating the palette.